source/core/slang-platform.cpp

```diff
@@ -170,10 +177,11 @@ SLANG_COMPILE_TIME_ASSERT(E_OUTOFMEMORY == SLANG_E_OUT_OF_MEMORY);
 /* static */SlangResult SharedLibrary::loadWithPlatformPath(char const* platformFileName, Handle& handleOut)
 {
     handleOut = nullptr;
-
-	void* h = dlopen(platformFileName, RTLD_NOW | RTLD_GLOBAL);
-	if(!h)
-	{
+    if (strlen(platformFileName) == 0)
```

Comment:
What does passing null or "" now mean?

Reply:
It means use RTLD_DEFAULT or load the function from the current process/module.